packages/app-mobile/components/screens/Note.tsx

```diff
@@ -1231,6 +1232,7 @@ const NoteScreen = connect((state: any) => {
 		themeId: state.settings.theme,
 		editorFont: [state.settings['style.editor.fontFamily']],
 		editorFontSize: state.settings['style.editor.fontSize'],
+		markdownToolbarEnabled: state.settings['editor.mobile.toolbarEnabled'],
```

Comment:
Maybe just "toolbarEnabled" for the prop too? At this point there's only one editor and even if we have a rich text one some day we'll probably use the same setting